package.json

```diff
@@ -12,8 +12,9 @@
   "types": "lib/index.d.ts",
   "exports": {
     ".": {
-      "import": "./lib/index.esm.js",
-      "require": "./lib/index.cjs.js"
+      "require": "./lib/index.cjs.js",
+      "node": "./lib/index.cjs.js",
```

Comment:
```suggestion
```

Unnecessary imo.

Reply:
You can try it out. It's the only way to make it work with `import('vue-demi')`